R/say.r

```diff
@@ -143,9 +147,15 @@ say <- function(what="Hello world!", by="cat", type="message", length=18,
       jsonlite::fromJSON(
         paste0('http://api.chrisvalleskey.com/fillerama/get.php?count=1&format=json&show=', what))$db$quote
   }
+  
+  if (!is.null(color)) {
+    color <- crayon::make_style(color)
+  } else {
+    color <- function(x) message(x)
+  }
 
   switch(type,
-         message = message(sprintf(who, what)),
+         message = message(cat(color(sprintf(who, what)))),
```

Comment:
hmm, does crayon still work if we don't use `cat()` here? idea with `message()` is that a user can still suppress a message with `supressMessages()`  - but that doesn't work if there's a `cat()` inside `message()`

Reply:
Actually yes it does! Good point, I'll remove the `cat`